packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -934,15 +934,80 @@ See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/invocat
 Lambda functions automatically create a log group with the name `/aws/lambda/<function-name>` upon first execution with
 log data set to never expire.
 
-The `logRetention` property can be used to set a different expiration period.
+The `logRetention` property can be used to set a different expiration period. In addition to this, `Tags` applied to a lambda `Function` can be propagated to the associated `logGroup` by setting the `propagateTagsToLogGroup` property to true on the `Function` construct. `Tags` that don't already exist on the `logGroup` for the `Function` will be added as new `Tags` when `propagateTagsToLogGroup` is set to true. As an example, consider the following `Function` and assume that the associated `logGroup` has no `Tags`:
+
```

Comment:
I would also add an example of adding tags specifically to the log group or
excluding tags from the log group. i.e. `Tags.of(function).add('env', 'beta', {
    excludeResourceTypes: ['Custom::LogsRetention']})`

Reply:
Good idea. I've added an example of this in the README.